torch/csrc/dynamo/utils.h

```diff
@@ -14,5 +22,21 @@
 #endif
 
 namespace torch::dynamo {
+
+extern "C" {
+
+#endif // __cplusplus
+
+PyObject* new_random_object();
+PyObject* random_module();
+PyObject* random_getstate(PyObject* rng);
+void random_setstate(PyObject* rng, PyObject* state);
```

Comment:
Coment on the ownership model here?  Are these owned pointers being returned?

Reply:
Moved ownership comments from .cpp file to .h file.